app_server/posts/models.py

```diff
@@ -16,6 +16,7 @@ class Post(models.Model):
     title = models.CharField(max_length=200)
     link = models.URLField(max_length=200, blank=True, unique=False)
     tags = TaggableManager(blank=True)
```

Comment:
We should update the `tags` here to `Foreignkey` to `TagDescription` model. Otherwise we would have multiple tag values for a post.

Reply:
Taggable manager does not use tag.description model. Tag description model is using taggable manager which appoint the values on tag. the user tests are succesful. Please carry on. If there is anything to debug we can check again in a group work.